cap-ci/jobs/cf-acceptance-tests-brain.tmpl

```diff
@@ -0,0 +1,51 @@
+{{ define "jobs_cf-acceptance-tests-brain" }}
```

Comment:
Yes, because when you template a resource, you want to prefix it with the type of resource. In this PR for example, you'd have a collision with `scripts_stratos` and `jobs_stratos` if you took away the prefix